packages/contracts-bedrock/test/L1/L2OutputOracle.t.sol

```diff
@@ -456,40 +405,69 @@ contract L2OutputOracleUpgradeable_Test is CommonTest {
-        assertEq(oracleImpl.submissionInterval(), deploy.cfg().l2OutputOracleSubmissionInterval());
-        assertEq(oracleImpl.L2_BLOCK_TIME(), deploy.cfg().l2BlockTime());
-        assertEq(oracleImpl.l2BlockTime(), deploy.cfg().l2BlockTime());
-        assertEq(oracleImpl.FINALIZATION_PERIOD_SECONDS(), deploy.cfg().finalizationPeriodSeconds());
-        assertEq(oracleImpl.finalizationPeriodSeconds(), deploy.cfg().finalizationPeriodSeconds());
-        assertEq(oracleImpl.PROPOSER(), deploy.cfg().l2OutputOracleProposer());
-        assertEq(oracleImpl.proposer(), deploy.cfg().l2OutputOracleProposer());
-        assertEq(oracleImpl.CHALLENGER(), deploy.cfg().l2OutputOracleChallenger());
-        assertEq(oracleImpl.challenger(), deploy.cfg().l2OutputOracleChallenger());
+        assertEq(oracleImpl.SUBMISSION_INTERVAL(), 120);
+        assertEq(oracleImpl.submissionInterval(), 120);
+        assertEq(oracleImpl.L2_BLOCK_TIME(), 12);
+        assertEq(oracleImpl.l2BlockTime(), 12);
+        assertEq(oracleImpl.FINALIZATION_PERIOD_SECONDS(), 12);
+        assertEq(oracleImpl.finalizationPeriodSeconds(), 12);
+        assertEq(oracleImpl.PROPOSER(), address(0));
+        assertEq(oracleImpl.proposer(), address(0));
+        assertEq(oracleImpl.CHALLENGER(), address(0));
+        assertEq(oracleImpl.challenger(), address(0));
 
         assertEq(oracleImpl.startingBlockNumber(), 0);
-        assertEq(oracleImpl.startingTimestamp(), 0);
+        assertEq(oracleImpl.startingTimestamp(), block.timestamp);
     }
 
     /// @dev Tests that the proxy cannot be initialized twice.
     function test_initializeProxy_alreadyInitialized_reverts() external {
+        uint256 submissionInterval = deploy.cfg().l2OutputOracleSubmissionInterval();
+        uint256 l2BlockTime = deploy.cfg().l2BlockTime();
         uint256 startingBlockNumber = deploy.cfg().l2OutputOracleStartingBlockNumber();
         uint256 startingTimestamp = deploy.cfg().l2OutputOracleStartingTimestamp();
+        address proposer = deploy.cfg().l2OutputOracleProposer();
+        address challenger = deploy.cfg().l2OutputOracleChallenger();
+        uint256 finalizationPeriodSeconds = deploy.cfg().finalizationPeriodSeconds();
+
         vm.expectRevert("Initializable: contract is already initialized");
-        l2OutputOracle.initialize({ _startingBlockNumber: startingBlockNumber, _startingTimestamp: startingTimestamp });
+        l2OutputOracle.initialize({
+            _submissionInterval: submissionInterval,
+            _l2BlockTime: l2BlockTime,
+            _startingBlockNumber: startingBlockNumber,
+            _startingTimestamp: startingTimestamp,
+            _proposer: proposer,
+            _challenger: challenger,
+            _finalizationPeriodSeconds: finalizationPeriodSeconds
+        });
     }
 
     /// @dev Tests that the implementation contract cannot be initialized twice.
     function test_initializeImpl_alreadyInitialized_reverts() external {
```

Comment:
We have test coverage of this case in `packages/contracts-bedrock/test/vendor/Initializable.t.sol` so this test case can be removed